pkg/runtime/runtime.go

```diff
@@ -842,17 +861,28 @@ func (a *DaprRuntime) getTopicRoutes() map[string]string {
 	}
 
 	for _, s := range subscriptions {
-		topicRoutes[s.Topic] = s.Route
+		if _, ok := a.pubSubs[s.PubsubName]; !ok {
+			msg := fmt.Sprintf("cannot subscribe to pubsub %s - does not exist", s.PubsubName)
```

Comment:
nit. comma after cannot subscribe to pubsub. It would be good to add topic name here in the msg.

Reply:
This error prints out if the app lists a pubsub in the response to /dapr/subscribe that does not exist because a component for  it was not created.  I modified the message to make it more clear and to point to where the config/setup error may be.